docs/remote/devcontainerjson-reference.md

```diff
@@ -9,185 +9,18 @@ DateApproved: 8/4/2022
 ---
 # devcontainer.json reference
 
-A `devcontainer.json` file in your project tells Visual Studio Code (and other services and tools that support the format) how to access (or create) a **development container** with a well-defined tool and runtime stack. It's currently supported by the [Remote - Containers](https://aka.ms/vscode-remote/download/containers) extension and [GitHub Codespaces](https://github.com/features/codespaces).
+A `devcontainer.json` file in your project tells Visual Studio Code (and other services and tools that support the format) how to access (or create) a **development container** with a well-defined tool and runtime stack.
```

Comment:
We could put a cut-to-the-chase "TL;DR - This document has moved to the dev container spec site!" up at the top with a direct link to reduce the amount of reading.

Can the left nav also point to the new spot @gregvanl?

Reply:
Added that sentence to the top and cut down a bit more content.

Would also be interested in what @gregvanl thinks!

Reply:
@bamurtaugh @Chuxel Apologies, just saw this. Yep, we can easily redirect an existing left nav item to a different location or site. 
For example:
Python > Data Science jumps to Data Science > Overview
Containers > Develop with Kubernetes redirects to https://docs.microsoft.com/visualstudio/bridge/bridge-to-kubernetes-vs-code

Reply:
Perfect, thanks for confirming, Greg! Yes, once we merge this PR, let's redirect the [devcontainer.json page](https://code.visualstudio.com/docs/remote/devcontainerjson-reference) under the Remote TOC to https://containers.dev/implementors/json_reference/.

I still update the content on our devcontainer.json page in this PR in case someone has specifically saved or otherwise gets to this URL.